govcd/nsxt_edge_cluster.go

```diff
@@ -77,28 +76,48 @@ func (vdc *Vdc) GetAllNsxtEdgeClusters(queryParameters url.Values) ([]*NsxtEdgeC
 		return nil, fmt.Errorf("VDC must have ID populated to retrieve NSX-T edge clusters")
 	}
 
+	// Get all NSX-T Edge clusters that are accessible to an organization VDC. The 'orgVdcId'filter
```

Comment:
Not sure if GitHub is displaying me this comment in the wrong location, but IMHO it would be better placed in the function docs. I mean, like this:
```
// GetAllNsxtEdgeClusters retrieves all NSX-T Edge clusters that are accessible to an organization VDC.
// The 'orgVdcId'filter key must be set with the ID of the VDC for which we want to get available Edge Clusters for.
// orgVdcId==urn:vcloud:vdc:09722307-aee0-4623-af95-7f8e577c9ebc
func (vdc *Vdc) GetAllNsxtEdgeClusters(queryParameters url.Values) ([]*NsxtEdgeCluster, error) {
```